### ISSUE TITLE
Web component & React: On theme g90 & g100, focus on link is not visible to user.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4378]
## Detailed description

Issue:  On theme g90 & g100, focus on link is not visible to user.

## Additional information

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/dacfecce-48be-4dce-9b1b-acea941731ce)

**Comment by IgnacioBecerra** [comment]
Just tested this, there is a white outline around the link when using g90 and g100 now. Seems like the issue has been fixed, unless I'm not replicating it correctly.

**Comment by Praveen-sr** [comment]
@IgnacioBecerra White outline around the link is not correct, user with low vision cannot identify the link.
  It must be blue outline.

**Expected:**
![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/d1ec0baa-8c57-4a37-804e-db4ed0e23b08)

------------------------------------------------------------------------------------------------------------------------

We have another related issue here:
On White theme - tab/ shift tab, focus is shown for full page (confusing the user)

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/94e93447-3f34-4c30-ae07-f3ddf6c8d8e3)

**Comment by IgnacioBecerra** [comment]
Discussed this issue with @JennySanchez and agreed that the `focus` outline should remain as white on the darker themes according to the [Carbon color guidelines](https://www.carbondesignsystem.com/guidelines/color/usage).

There was a weird occurrence a few days ago where one of the bluemix React sites wasn't turning the `Quote` background black in G90/G100, which probably lead to the white focus on gray appearance that was being reported initially.

It now looks like the following:
![Screen Shot 2020-12-04 at 1.06.50 PM.png](https://images.zenhubusercontent.com/5f63c83b3584b9bd910ec115/19bdaee8-d863-498d-a265-32dfb37d7eef)

@Praveen-sr As for the second issue you brought up (full page focus), would you mind creating a new issue for that? Thank you!
